src/quiz.js

```diff
@@ -73,17 +75,17 @@ function Question() {
     //---------------------------------------------
     //This is to make the answers go green
 
-    this.makeGreen = function() {
-        questionContainer.className += " goGreen";
+    makeGreen() {
+        this.questionContainer.className += " goGreen";
     }
 
     //---------------------------------------------
     //This is the code to change to display all the answer options
 
-    this.setAnswerList = function (answerList) {
-        var answerElement1 = createAnswerContainer();
-        setText(answerElement1, answerList[0]);
-        setRadioButton(answerElement1, 'groupName', answerList[0]);
+    setAnswerList(answerList) {
+        var answerElement1 = this.createAnswerContainer();
```

Comment:
If we're doing classes we might as well do `const` and `let`

Reply:
I think we should move to `const` and `let` at some point. However based on the training the education centre staff have had so far I think we are best to stick to `var` which they are already familiar with. Also changing to `const` and `let` requires changes to the education centre handouts and additional work for them.